__tests__/unit/logic.test.ts

```diff
@@ -193,10 +224,12 @@ describe('TADProc Service', () => {
         const channelResult = requestBody.channelResult;
         const message = networkMap.messages.find((tran) => tran.txTp === transaction.TxTp);
 
-        const result = await handleExecute(requestBody);
-        expect(result).toBeDefined();
-        if (res?.report?.status) expect(res.report.status).toBe('ALRT');
-        else throw 'Test failed - expect response to be called';
+        if (message) {
```

Comment:
bad practice to put this guard in unit tests - if someone modifies the method that returns the networkmap to return null, the tests will no longer run and not report an error / failure of the test to run. 

If you need to add the null check, at least put a throw in the "else" of the guard. That would be fine.